CMakeLists.txt

```diff
@@ -232,9 +232,12 @@ if(USE_TENSORRT)
 endif()
 if(WITH_TENSORFLOW2_LIB)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DDLR_TENSORFLOW2")
-    include_directories("${WITH_TENSORFLOW2_LIB}/include")
+    if(NOT TENSORFLOW2_INCLUDE)
+        message(ERROR "TENSORFLOW2_INCLUDE is required.")
```

Comment:
This message stmt will print the following and continue cmake
```
ERRORTENSORFLOW2_INCLUDE is required.
```
CMake doc - https://cmake.org/cmake/help/latest/command/message.html
```
FATAL_ERROR
CMake Error, stop processing and generation.

SEND_ERROR
CMake Error, continue processing, but skip generation.
```
Lets fix TRT msg too (line 217)
```
message(ERROR "Could not find TensorRT.")
```

Reply:
Fixed!